docs/CONFIG-VARS.md

```diff
@@ -14,8 +14,8 @@ Supported configuration variables are listed in the table below.  All variables
     - [Default Nodepool](#default-nodepool)
     - [Additional Nodepools](#additional-nodepools)
   - [Storage](#storage)
-    - [storage_type=standard - nfs server VM](#storage_typestandard---nfs-server-vm)
-    - [storage_type=ha - Azure NetApp](#storage_typeha---azure-netapp)
+    - [NFS Server VM (only when `storage_type=standard`)](#nfs-server-vm-only-when-storage_typestandard)
```

Comment:
Maybe leave the stuff in the brackets out of the TOC, so the TOC is cleaner i. e.

 - [NFS Server VM](#nfs-server-vm-only-when-storage_typestandard)
 - [Azure NetApp](#azure-netapp-files-only-when-storage_typeha)